packages/sdk-android/src/runner.ts

```diff
@@ -518,4 +525,7 @@ export const runAndroidLog = async (c: Context) => {
     return child.then((res) => res.stdout).catch((err) => Promise.reject(`Error: ${err}`));
 };
 
+export const isString = (target: boolean | string): boolean => {
```

Comment:
why export it from `runner`? Either don't export it and prefix it with _, so we know it's private or move this to some utils and import it from there.